src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -279,6 +280,16 @@ private void AddArtistTokens(Dictionary<string, Func<TokenMatch, string>> tokenH
             }
         }
 
+        private void AddTrackArtistTokens(Dictionary<string, Func<TokenMatch, string>> tokenHandlers, List<Track> tracks)
+        {
+            var artists = tracks.Select(t => t.ArtistMetadata.Value)
```

Comment:
What's the difference between `Artist` and `ArtistMetadata` in the data model? It looks like `Artist` is still "Various Artists" for these tracks, but `ArtistMetadata` is the correct artist.

Reply:
Artist is the actual record that we follow in the db (monitored or not, profiles are attached to, etc...) ArtistMetadata is basically the data record for a given artist, it can be attached to an artist object or a track object.